bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -180,6 +182,24 @@ public void run() {
     connectionStartupExecutor.shutdown();
   }
 
+  private static void enableTracing(AbstractManagedChannelImplBuilder<?> builder) {
+    Method setEnableTracingMethod = null;
+    try {
+      setEnableTracingMethod =
+          AbstractManagedChannelImplBuilder.class.getMethod("setEnableTracing", boolean.class);
+    } catch (NoSuchMethodException | SecurityException e1) {
+      return;
+    }
+    if (setEnableTracingMethod != null) {
+      try {
+        setEnableTracingMethod.invoke(builder, true);
+        System.out.println("ENABLING TRACING!!!!");
```

Comment:
Please remove debug code & add a comment mentioning when this method came to be in grpc and when its no longer needed